bench/irmin-pack/encoding/base58.ml

```diff
@@ -0,0 +1,147 @@
+(*****************************************************************************)
```

Comment:
Maybe move this into Irmin core directly (so it's easy to use base58 for other use-cases).

Reply:
hum actually maybe scratch this as it uses zarith (and I'm not sure we want that dependency yet).

Reply:
May be a good opportunity to integrate it into `ocaml-base64` which is transitively a dependence of `irmin-*` in anyway.

Reply:
It's unfortunately a different logic than in base64 as it's not byte aligned: it builds a large integer (with Zarith) then serialise it